src/iidm/VariantManager.cpp

```diff
@@ -76,12 +76,14 @@ void VariantManager::cloneVariant(const std::string& sourceVariantId, const std:
     }
 
     if (!recycled.empty()) {
+        m_network.allocateVariantArrayElement(recycled, sourceIndex);
```

Comment:
Network is not is index (unlike in java), so call handles manually.
Another solution may be to add a ref to the network in NetworkIndex and to update `NetworkIndex::getAll<Identifiable*` so that it pushes a reference to the network also, but it wasn't trivial, so I prefered to patch manually (simpler solution) to make PR reviewable quicker.

Reply:
Could you explain why it's not trivial? If the network is not in the index, the export of the network extensions won't work as well (maybe other impacts)